### ISSUE TITLE
Doc Usage: Progress

### ISSUE THREAD

**Issue by francisrupert** [USDA-FSA/fsa-design-system#294]
### Background

* [Progress Component](http://usda-fsa.github.io/fsa-design-system/components/progress/) was made available in `fsa-style@2.4.0`, via https://github.com/USDA-FSA/fsa-style/issues/356
* Currently has MVP level of documentation (via #278).

### Task

- [x] Documentation Research
- [x] Variations
- [x] **Usage** 
- [x] Accessibility implications
- [x] JavaScript implications
- [x] Related Resources

**Comment by bignamehere** [comment]
@francisrupert As part of the research on the Progress component, I had a thought about a11y. There doesn't seem to be anything implemented in regards to role. Since this component (and splash) are mainly shown as visual representations, are we covered from an a11y standpoint?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role

Should the above be implemented within the ARIA of the components?

**Comment by bignamehere** [comment]
Closing issue, but a11y details need to be reviewed and "role" details should be added.
